recipes/iiboost/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "iiboost" %}
+{% set version = "0.2" %}
+{% set sha256 = "0ecd906b453a6a278651e324352bd6f820b70aa4ca15fa42843bdb5c01ee7079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/cbecker/iiboost/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    - cmath_fixes.diff
+    - python3.diff
+    - msvc_fixes.diff
```

Comment:
Please submit these patches upstream.

Also patching in Python 3 support for Python 2 only libraries has been frowned upon by other reviewers and core members in the past. Though I get that being stuck on an old Python because of a dependency is a bit of a drag especially for a few lines of code. Though I think we will have a good argument for doing this if upstream adds Python 3 support into `master` (potentially using a variant of this patch). Let's see what they say.